templates/boards/welcome.html

```diff
@@ -2,5 +2,32 @@
 {% load compress mf_extras static %}
 
 {% block content %}
-  <h1>welcome to this site</h1>
-{% endblock %}
+<div class="container">
+  <div class="row mt-5 justify-content-center">
+    <div class="col-auto">
+      <h1>Make Managing Simple</h1>
+    </div>
+  </div>
+  <div class="row mt-5">
+    <div class="col-6 col-md-3 border-right">
+      <div class="p-3">
+        <h2>Updates</h2>
+        <p>sometext sometext sometext sometext sometext sometext sometext sometext sometext sometext sometext sometext
+        </p>
+        <p><a href="#" target="_blank" class="btn btn-success">Learn More &raquo;</a></p>
+      </div>
+    </div>
+    <div class="col-6 col-md-3">
+      <div class="p-3">
+        <h2>Updates</h2>
+        <p>sometext sometext sometext sometext sometext sometext sometext sometext sometext sometext sometext sometext
+        </p>
+        <p><a href="#" target="_blank" class="btn btn-success">Learn More &raquo;</a></p>
+      </div>
+    </div>
+    <div class="col-12 col-md-6">
+      <img src="{% static 'img/front.jpg' %}" class="mt-3 img-fluid" alt="Responsive image">
+    </div>
+  </div>
+</div>
+{% endblock %}LoginLogin
```

Comment:
Maybe you can replace the picture with a signup button and add a full-screen background picture?